sql/step_a110_update_operations_table_change_ope_for_to_enum.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE OH_OPERATION MODIFY COLUMN OPE_FOR VARCHAR(20);
+
+UPDATE OH_OPERATION
+    SET OPE_FOR='OPD_ADMISSION' WHERE OPE_FOR='1';
+UPDATE OH_OPERATION
+    SET OPE_FOR='ADMISSION' WHERE OPE_FOR='2';
+UPDATE OH_OPERATION
+    SET OPE_FOR='OPD' WHERE OPE_FOR='3';
+
+ALTER TABLE OH_OPERATION
+    MODIFY COLUMN OPE_FOR ENUM('OPD', 'ADMISSION', 'OPD_ADMISSION') DEFAULT 'OPD_ADMISSION';
```

Comment:
Let's use lowercase values, order matters.

```suggestion
ALTER TABLE OH_OPERATION MODIFY COLUMN OPE_FOR VARCHAR(10);

UPDATE OH_OPERATION
    SET OPE_FOR='opd_adm' WHERE OPE_FOR='1';
UPDATE OH_OPERATION
    SET OPE_FOR='adm' WHERE OPE_FOR='2';
UPDATE OH_OPERATION
    SET OPE_FOR='opd' WHERE OPE_FOR='3';

ALTER TABLE OH_OPERATION
    MODIFY COLUMN OPE_FOR ENUM('opd_adm','adm', 'opd', ) DEFAULT 'opd_adm';
```

Reply:
NOTE: we are using ENUM for the first time. It is much more efficient than VARCHAR, so we may consider to refactor similar cases... @dbmalkovsky wdyt?